packages/frontend/src/PortalMap/ControlBar/DeleteNode.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { FC, useCallback, useState } from 'react'
+
+import {
+  Button,
+  Dialog,
+  DialogActions,
+  DialogTitle,
+  IconButton,
+} from '@material-ui/core'
+import DeleteIcon from '@material-ui/icons/Delete'
+
+import { CytoEdgeData } from '../'
+import useDeleteZone from '../hooks/useDeleteZone'
+import styles from './styles.module.scss'
+
+interface DeleteNodeProps {
+  edgeData: CytoEdgeData[]
+  zoneName: string | undefined
+}
+
+const DeleteNode: FC<DeleteNodeProps> = ({ edgeData, zoneName }) => {
+  const deletePortals = useDeleteZone()
+  const [isOpen, setIsOpen] = useState<boolean>(false)
+
+  const handleClose = useCallback(
+    (deleteZone: boolean) => {
+      if (deleteZone) {
+        deletePortals(edgeData)
+      }
+
+      setIsOpen(false)
+    },
+    [deletePortals, edgeData]
+  )
+
+  return edgeData.length > 0 ? (
```

Comment:
Use `edgeData.length > 0 &&` ?

Reply:
Oops yeah. I had just a falsy check and was printing a 0. I was getting tired and it looks like I fixed the problem 2 different ways.

Reply:
![Screen Shot 2020-11-29 at 5 14 38 PM](https://user-images.githubusercontent.com/2829924/100554930-648dce00-3266-11eb-9b2f-7eb903de805d.png)
Actually this is why it's not that.